web/save.go

```diff
@@ -107,27 +106,29 @@ func checkAndSave(request *http.Request) string {
 		dnsConf.DNS.Name = v.DnsName
 		dnsConf.DNS.ID = strings.TrimSpace(v.DnsID)
 		dnsConf.DNS.Secret = strings.TrimSpace(v.DnsSecret)
+
+		if v.Ipv4Enable && v.Ipv4Domains == "" {
+			util.Log("IPv4 Domains 为空")
+		}
+
 		dnsConf.Ipv4.Enable = v.Ipv4Enable
 		dnsConf.Ipv4.GetType = v.Ipv4GetType
 		dnsConf.Ipv4.URL = strings.TrimSpace(v.Ipv4Url)
 		dnsConf.Ipv4.NetInterface = v.Ipv4NetInterface
 		dnsConf.Ipv4.Cmd = strings.TrimSpace(v.Ipv4Cmd)
-		if strings.Contains(v.Ipv4Domains, "\r\n") {
-			dnsConf.Ipv4.Domains = strings.Split(v.Ipv4Domains, "\r\n")
-		} else {
-			dnsConf.Ipv4.Domains = strings.Split(v.Ipv4Domains, "\n")
+		dnsConf.Ipv4.Domains = splitLines(v.Ipv4Domains)
+
+		if v.Ipv6Enable && v.Ipv6Domains == "" {
```

Comment:
```if v.Ipv4Domains == "" && v.Ipv6Domains == "" {...}``` Because it is enabled by default, this can reduce unnecessary prompts

Reply:
> `if v.Ipv4Domains == "" && v.Ipv6Domains == "" {...}` Because it is enabled by default, this can reduce unnecessary prompts

Yes, we should only deal with first time configuration.